docs/pages/summaries.md

```diff
@@ -57,6 +57,27 @@ variation_groups:
         variation_description:
           Adding the `o-summary__mobile` modifier makes the summary
           behavior only show up on a mobile (narrow) page width.
+      - variation_name: Summary Minimal
+        variation_code_snippet: >-
+          <div class="o-summary-minimal">
+            <div class="o-summary-minimal_content">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+            </div>
+
+            <button class="o-summary-minimal_btn">
+                <span class="o-summary-minimal_cue-open">
+                    Show full list of relevant parties
```

Comment:
If we're making a component out of this, this text should be customizable or at least more general.